scripts/fix-uninstallable.py

```diff
@@ -0,0 +1,127 @@
+# check all revisions in the lockfile if they are installable
+# remove if not and the next installable version is installed
```

Comment:
But this script is not installing, isn't it? will it add the next proper version to the yaml file?